addons/website_slides/static/src/js/slides.js

```diff
@@ -20,7 +20,7 @@ publicWidget.registry.websiteSlides = publicWidget.Widget.extend({
             // if presentation 7 days, 24 hours, 60 min, 60 second, 1000 millis old(one week)
             // then return fix formate string else timeago
             var displayStr = '';
-            if (datetimeObj && new Date().getTime() - datetimeObj.getTime() > 7 * 24 * 60 * 60 * 1000) {
+            if (datetimeObj && new Date().getTime() - datetimeObj.valueOf() > 7 * 24 * 60 * 60 * 1000) {
                 displayStr = DateTime.fromJSDate(datetimeObj).toFormat('DD');
```

Comment:
As @sakesun noticed, these calls to `fromJSDate` are now obsolete too.

Reply:
Just pushed an update to this fix.
A big thanks to @sakesun :+1: